src/holofuel/pages/TransactionHistory/TransactionHistory.js

```diff
@@ -58,58 +62,113 @@ function useFetchCounterparties () {
   }
 }
 
-// Display - Functional Components with Hooks :
+// returns an array of objects of the form { label, transactions }, sorted chronologicaly
+function partitionByDate (transactions) {
+  const now = moment()
+  const today = now.clone().startOf('day')
+  const yesterday = now.clone().subtract(1, 'days').startOf('day')
+
+  const isToday = momentDate => momentDate.isSame(today, 'd')
+  const isYesterday = momentDate => momentDate.isSame(yesterday, 'd')
+
+  const getLabel = ({ timestamp }) => {
+    const momentDate = moment(timestamp)
+    if (isToday(momentDate)) return 'Today'
+    if (isYesterday(momentDate)) return 'Yesterday'
+    return momentDate.format('MMMM Do')
+  }
```

Comment:
When considering the timestamp label approach for a similar need in Inbox, I went ahead and wrote this logic into the holofuel interface and returned an additional entry for this label in the transaction gql type, in order to avoid nesting that data into a page or component.  This makes it much easier for all pages to readily have all the data they need to sort through the transactions as done previously.  Take a look and let's discuss whether we'd like to reference that data here too.